source/index.html.md

```diff
@@ -1,164 +1,644 @@
 ---
-title: API Reference
+title: Elemeno Documentation
 
 language_tabs:
-  - rest
-  - js
-  - php
+  - http
 
 includes:
-  - errors
 
 search: false
 ---
 
 # Introduction
 
-Welcome to the Kittn API! You can use our API to access Kittn API endpoints, which can get information on various cats, kittens, and breeds in our database.
+NEED SOME STUFF HERE
 
-We have language bindings in Shell, Ruby, and Python! You can view code examples in the dark area to the right, and you can switch the programming language of the examples with the tabs in the top right.
+# Single Items
 
-This example API documentation page was created with [Slate](https://github.com/tripit/slate). Feel free to edit it and use it as a base for your own API's documentation.
+Single Items are items that standalone and do not belong to a collection. They have their own unique content model, and the item is returned via the API as a single resource. Single items are perfect for things like an "About Us" or "Contact" pages on a website, or the landing screen of an application. Any content that will exist on it's own can be thought of as a Single Item. 
```

Comment:
- `standalone` would be two words, `stand alone`, in this instance.
- `it's` in the last sentence should be `its`